CONTRIBUTING.md

```diff
@@ -74,8 +70,58 @@ BREAKING CHANGE: numpy 1.0.2 no longer supported
 Refs: #123
 ```
 
-Code
-------
+### Breaking changes and deprecation
+
+Since we are still pre-1.0, [SemVer] states that any release may contain breaking
+changes. However, breaking changes should not be made without warning.
+
+**Any breaking changes must have a deprecation period of at least one minor release**
```

Comment:
Do we want to put a time period in weeks on these? We do have some long gaps between releases. Waiting for two releases might encourage an arbitrary extra release as a box ticking exercise.

Reply:
We have just agreed `the longer of one minor release or one month` and double that for large changes.